### ISSUE TITLE
sls deploy -v no longer working

### ISSUE THREAD

**Issue by CORT0619** [nya1/serverless-ignore#6]
After installing serverless-ignore, adding serverless-ignore to my plugins,  and adding a .slsignore my code no longer deploys to aws cloudformation when i run `sls deploy -v`. I receive no errors literally nothing happens. After commenting out that plugin everything works as expected. I'm on version 2.48.0.

![image (1)](https://user-images.githubusercontent.com/17831460/123098403-1ce7b300-d3ff-11eb-8bc5-4ff3d2cf48a1.png)

**Comment by charlie632** [comment]
Same thing happening on my end..

Doing `sls deploy -s development` and I get some deprecation warnings and that's it.

`serverless-ignore` version: ^0.1.6


```
Framework Core: 2.51.2
Plugin: 5.4.3
SDK: 4.2.3
Components: 3.13.3

```

**Comment by renatoalencar** [comment]
I had the same issue here, the problem seems to be the deploy command configuration of the plugin. I don't really know how this works, but removing `package` from the `lifecycleEvents` array works, I forked the plugin code to an internal implementation on a directory `plugins` and used it like this on my `serverless.yml` file.

```yaml
plugins:
  - ./plugins/serverless-ignore
```

**Comment by nya1** [comment]
fixed in 0.2.1 version
